README.md

```diff
@@ -42,3 +40,9 @@ For a release build, you can also run `yarn release` for a much smaller applicat
    - If you added the extension as a `Case home page`:
       1. Navigate to the Case Home page.
       1. Click the extension tab in the navigation pane.
+
+## How to deploy it
```

Comment:
Is this just for the release build? If so, you might want to clarify. E.g. "How to deploy it to production"

Reply:
Ah, good point.

Reply:
I pushed this change up out-of-band. You should see it now on the front page of this repository.